easybuild/easyconfigs/l/libQGLViewer/libQGLViewer-2.8.0-GCCcore-10.3.0.eb

```diff
@@ -14,7 +14,10 @@ dependencies = [
     ('libGLU', '9.0.1'),
 ]
 
-builddependencies = [('Qt5', '5.15.2')]
+builddependencies = [
+('Qt5', '5.15.2'),
```

Comment:
wait, i didn't spot this the first time around, is Qt5 really just a builddep for this software???

If you have it build already, please check `ldd libQGLViewer*.so`

Reply:
```
[satishk@tcn1 QGLViewer]$ ldd libQGLViewer-qt5.so
	linux-vdso.so.1 (0x00007ffcf372f000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x000015090c1c7000)
	libGL.so.1 => /sw/arch/Centos8/EB_production/2021/software/libglvnd/1.3.3-GCCcore-10.3.0/lib/libGL.so.1 (0x000015090c4c4000)
	libGLU.so.1 => /sw/arch/Centos8/EB_production/2021/software/libGLU/9.0.1-GCCcore-10.3.0/lib/libGLU.so.1 (0x000015090c109000)
	libQt5OpenGL.so.5 => /sw/arch/Centos8/EB_production/2021/software/Qt5/5.15.2-GCCcore-10.3.0/lib/../lib64/libQt5OpenGL.so.5 (0x000015090c45f000)
	libQt5Widgets.so.5 => /sw/arch/Centos8/EB_production/2021/software/Qt5/5.15.2-GCCcore-10.3.0/lib/../lib64/libQt5Widgets.so.5 (0x000015090b98f000)
	libQt5Gui.so.5 => /sw/arch/Centos8/EB_production/2021/software/Qt5/5.15.2-GCCcore-10.3.0/lib/../lib64/libQt5Gui.so.5 (0x000015090b215000)
	libQt5Xml.so.5 => /sw/arch/Centos8/EB_production/2021/software/Qt5/5.15.2-GCCcore-10.3.0/lib/../lib64/libQt5Xml.so.5 (0x000015090b1c9000)
	libQt5Core.so.5 => /sw/arch/Centos8/EB_production/2021/software/Qt5/5.15.2-GCCcore-10.3.0/lib/../lib64/libQt5Core.so.5 (0x000015090ab60000)
	libstdc++.so.6 => /gpfs/admin/_hpc/sw/arch/AMD-ZEN2/Centos8/EB_production/2021/software/GCCcore/10.3.0/bin/../lib/gcc/x86_64-pc-linux-gnu/10.3.0/../../../../lib64/libstdc++.so.6 (0x000015090a979000)
	libm.so.6 => /lib64/libm.so.6 (0x000015090a5f7000)
	libgcc_s.so.1 => /gpfs/admin/_hpc/sw/arch/AMD-ZEN2/Centos8/EB_production/2021/software/GCCcore/10.3.0/bin/../lib/gcc/x86_64-pc-linux-gnu/10.3.0/../../../../lib64/libgcc_s.so.1 (0x000015090c441000)
	libc.so.6 => /lib64/libc.so.6 (0x000015090a232000)
	/lib64/ld-linux-x86-64.so.2 (0x000015090c3e7000)
	libGLdispatch.so.0 => /sw/arch/Centos8/EB_production/2021/software/libglvnd/1.3.3-GCCcore-10.3.0/lib/libGLdispatch.so.0 (0x000015090a17a000)
	libGLX.so.0 => /sw/arch/Centos8/EB_production/2021/software/libglvnd/1.3.3-GCCcore-10.3.0/lib/libGLX.so.0 (0x000015090a144000)
	libpng16.so.16 => /sw/arch/Centos8/EB_production/2021/software/libpng/1.6.37-GCCcore-10.3.0/lib64/libpng16.so.16 (0x000015090a0eb000)
	libz.so.1 => /sw/arch/Centos8/EB_production/2021/software/zlib/1.2.11-GCCcore-10.3.0/lib/libz.so.1 (0x000015090c418000)
	libharfbuzz.so.0 => /lib64/libharfbuzz.so.0 (0x0000150909e45000)
	libdouble-conversion.so.3 => /sw/arch/Centos8/EB_production/2021/software/double-conversion/3.1.5-GCCcore-10.3.0/lib64/libdouble-conversion.so.3 (0x0000150909e28000)
	libicui18n.so.60 => /lib64/libicui18n.so.60 (0x0000150909967000)
	libicuuc.so.60 => /lib64/libicuuc.so.60 (0x00001509095a5000)
	libicudata.so.60 => /lib64/libicudata.so.60 (0x00001509079fb000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00001509077f7000)
	libpcre2-16.so.0 => /sw/arch/Centos8/EB_production/2021/software/PCRE2/10.36-GCCcore-10.3.0/lib64/libpcre2-16.so.0 (0x0000150907753000)
	libzstd.so.1 => /sw/arch/Centos8/EB_production/2021/software/zstd/1.4.9-GCCcore-10.3.0/lib/libzstd.so.1 (0x0000150907605000)
	libgthread-2.0.so.0 => /sw/arch/Centos8/EB_production/2021/software/GLib/2.68.2-GCCcore-10.3.0/lib64/libgthread-2.0.so.0 (0x0000150907601000)
	libglib-2.0.so.0 => /sw/arch/Centos8/EB_production/2021/software/GLib/2.68.2-GCCcore-10.3.0/lib64/libglib-2.0.so.0 (0x000015090746e000)
	libX11.so.6 => /sw/arch/Centos8/EB_production/2021/software/X11/20210518-GCCcore-10.3.0/lib64/libX11.so.6 (0x00001509072ea000)
	libfreetype.so.6 => /sw/arch/Centos8/EB_production/2021/software/freetype/2.10.4-GCCcore-10.3.0/lib/../lib64/libfreetype.so.6 (0x00001509071f5000)
	libgraphite2.so.3 => /lib64/libgraphite2.so.3 (0x0000150906fc7000)
	libxcb.so.1 => /sw/arch/Centos8/EB_production/2021/software/X11/20210518-GCCcore-10.3.0/lib/libxcb.so.1 (0x0000150906f99000)
	libbz2.so.1.0 => /sw/arch/Centos8/EB_production/2021/software/bzip2/1.0.8-GCCcore-10.3.0/lib64/libbz2.so.1.0 (0x0000150906f75000)
	libXau.so.6 => /sw/arch/Centos8/EB_production/2021/software/X11/20210518-GCCcore-10.3.0/lib/libXau.so.6 (0x0000150906f6d000)
	libXdmcp.so.6 => /sw/arch/Centos8/EB_production/2021/software/X11/20210518-GCCcore-10.3.0/lib/libXdmcp.so.6 (0x0000150906f64000)
[satishk@tcn1 QGLViewer]$ ldd libQGLViewer-qt5.so.2
```

Reply:
Well, this is what I kind of expected. 

I skimmed through the source code to be sure, but i didn't find that this code depended on anything other than Qt and GLU, so i think all the suspicious system linked things:

```
	libharfbuzz.so.0 => /lib64/libharfbuzz.so.0 (0x0000150909e45000)
	libicui18n.so.60 => /lib64/libicui18n.so.60 (0x0000150909967000)
	libicuuc.so.60 => /lib64/libicuuc.so.60 (0x00001509095a5000)
	libicudata.so.60 => /lib64/libicudata.so.60 (0x00001509079fb000)
	libgraphite2.so.3 => /lib64/libgraphite2.so.3 (0x0000150906fc7000)
```
are inherited from your Qt5 module (which means that version of Qt5 should have added HarfBuzz, ICU, graphite2 as dependencies, but that's not libQGLViewer's fault.)
At least some of that seems to have been adressed in newer Qt5 easyconfigs.

So, this package should have Qt5 and libGLU as deps.

Reply:
Have moved `Qt5` from build dependencies to dependencies.